README.md

```diff
@@ -76,7 +76,7 @@ Transaction hash: 0x7a6d29b281aaef31e5d7ae5080b8800614f4f053b63fbb83a0d870807197
 ```shell
 PASSKEY_ACCOUNT_FACTORY_ADDRESS="<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>"
 
-forge verify-contract --watch --chain "sepolia" --verifier "etherscan" --etherscan-api-key ${ETHERSCAN_API_KEY} --compiler-version ${COMPILER_VERSION} --constructor-args $(cast abi-encode "constructor(address)" ${ENTRYPOINT_ADDRESS}) ${PASSKEY_ACCOUNT_FACTORY_ADDRESS} "src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory"
+forge verify-contract --watch --chain ${NETWORK_NAME} --verifier "etherscan" --etherscan-api-key ${ETHERSCAN_API_KEY} --compiler-version ${COMPILER_VERSION} --constructor-args $(cast abi-encode "constructor(address)" ${ENTRYPOINT_ADDRESS}) ${PASSKEY_ACCOUNT_FACTORY_ADDRESS} "src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory"
```

Comment:
According to the [doc](https://book.getfoundry.sh/reference/forge/forge-verify-contract#forge-verify-contract), chain can be assigned by id. In my opinion it could be better to use id instead of name.